TheRookiesApp/PageObject/MenuLeft.cs

```diff
@@ -0,0 +1,25 @@
+﻿using CoreFramework.DriverCore;
+
+namespace TheRookiesApp.PageObject;
+
+public class MenuLeft : WebDriverBase
+{
+
+    public MenuLeft() : base()
+    {
+
+    }
+
+    //public bool IsCorrectRedirect()
+    //{
+    //    if (Constant.BASE_URL + addedURL == GetUrl())
+    //    {
+    //        return true;
+    //    }
+    //    else
+    //    {
+    //        return false;
+    //    }
+    //}
```

Comment:
Remove all unused code. Please check again for the other files

Reply:
Done